assignment-1/src/app/colour-pallete-container/colour-pallete-container.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="colour-pallete-container">
+    <button class="colour-button clr1" value="#8f8f8f" (click)="onColorChange($event)" [ngClass]="{'selected':currentTheme==='#8f8f8f'}"></button>
```

Comment:
we don't need a value attribute for this, this can be directly passed as input

Reply:
value was passed to the function